### PR TITLE
Correction of few typo errors (conf/)

### PR DESCRIPTION
**Pull request by Zopokx**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Just some typo errors in /conf/ that should be fixed.
Also, one improvement in understanding atcommand_mobinfo_type config (gm.conf).

### CODE REVIEW

**Comment by 4144** [comment]
why you closed previous pr and create same new?

**Comment by Zopokx** [comment]
Sorry for that @4144, I've lost a lot of practice and still lack git experience. Is everything ok or is it a problem?

**Comment by 4144** [comment]
need squash commits ``Fix typo error & space replacement in battlegrounds.conf`` and ``Fixing the fix in previous commit Y_Y``

you can use for this interactive rebase or other way.

**Comment by Zopokx** [comment]
Well, it seems that it is ready now, isn't it?
I hope I'm not wrong ...  (Thanks for the tip!)

File: doc/constants.md [hunk 1]
```diff
@@ -3657,7 +3657,7 @@
 - `MER_SWORDMAN09`: 6045
 - `MER_SWORDMAN10`: 6046
 
-### checkquest
+### questprogress
```
Comment (Kenpachi2k13) [inline doc/constants.md:3660]: This file automatically generated. Change it in [db/constants.conf](https://github.com/HerculesWS/Hercules/blob/stable/db/constants.conf#L3759) instead.

File: AUTHORS [hunk 2]
```diff
@@ -214,6 +213,7 @@ Yommy <yomanda@gmail.com>
 Your Name <ibrahem.h.basyone@gmail.com>
 zackdreaver <zackdreaver@gmail.com>
 Zarbony <justonlinegames@web.de>
+Zopokx <zopokx@gmail.com>
```
Comment (4144) [inline AUTHORS:216]: for rename commit authors need add rename string into ``.mailmap``
then generate authors list by script ``tools/authors.sh`` and replace first list of authors in AUTHORS file

File: AUTHORS [hunk 3]
```diff
@@ -186,7 +186,6 @@ Shido <the.keikun@gmail.com>
 Shizuke <matiasdvt@hotmail.com>
 sigtus <vantimothy@openmailbox.org>
 sirius <fw@f-ws.de>
-Sirius <zopokx@gmail.com>
```
Comment (Kenpachi2k13) [inline AUTHORS:189]: The _AUTHORS_ files is generated from commit logs. If you want to "rename" yourself please use the [.mailmap](https://github.com/HerculesWS/Hercules/blob/stable/.mailmap) file.

**Comment by MishimaHaruna** [comment]
I addressed the remaining open points in a new commit and pushed it to the PR branch

**Pull request merged**